.stylelintrc.js

```diff
@@ -1,6 +1,25 @@
 module.exports = {
   extends: '@wagtail/stylelint-config-wagtail',
   rules: {
+    'no-invalid-position-at-import-rule': [
+      true,
+      {
+        ignoreAtRules: ['tailwind'],
+      },
+    ],
```

Comment:
It looks like stylelint is still failing – if configuring the rule like that doesn’t work, we could also disable that rule just within that one file.